docs/reference/mpi_engine.rst

```diff
@@ -0,0 +1,6 @@
+The Globus MPI Engine
+===========================
```

Comment:
Branding wise, that should be `The Globus Compute MPI Engine`, no?

Reply:
Now that you've pointed it out, it sounds weird. But at the same time `GlobusComputeMPIEngine` is a mouthful.

Reply:
We have consensus that we'll keep the same name.